R/indexing.R

```diff
@@ -425,28 +424,28 @@ expand.grid.unordered <- function(i, j, loops = FALSE, directed = FALSE) {
   ## Argument checks
   if ((!missing(from) || !missing(to)) &&
     (!missing(i) || !missing(j))) {
-    stop("Cannot give 'from'/'to' together with regular indices")
+    cli::cli_abort("Cannot use {.arg from}/{.arg to} together with regular indices")
   }
   if ((!missing(from) && missing(to)) ||
     (missing(from) && !missing(to))) {
-    stop("Cannot give 'from'/'to' without the other")
+    cli::cli_abort("Cannot use {.arg from}/{.arg to} without the other")
```

Comment:
a terrifying though: do we need periods at the end of error messages?

Reply:
oh god you opened pandoras box :D. But (unfortunately) a good point. Not sure what the "state-of-the-art" is but we should probably make this consistent (eventually)

Reply:
"a good point" what a pun :joy: I'll open an issue

Reply:
hahaha true, but that was an accidental pun :D